docs/user-guide/wysiwyg/README.md

```diff
@@ -157,19 +166,24 @@ You can embed specific attributes of work packages or projects using the followi
 </div>
 
 
-
+### Embedding attribute help texts
 You can also embed attribute values and [their help texts](../../system-admin-guide/attribute-help-texts/) by using `workPackageLabel` instead: `workPackageLabel:1234:status` would output the translated label for "Status" and (if exists), the corresponding help text for it.
```

Comment:
Is this also possible for projectLabel ?

Reply:
yes, it is. This can be seen in the second table below. I'll add a sentence here to make this clear